getting-started/bare-metal/about.md

```diff
@@ -1,56 +1,53 @@
 
-When {{site.prodname}} networking is enabled...
+| Install {{site.prodname}} with... | Requires                         | Use case                                                     | Supported install methods                      |
+| --------------------------------- | -------------------------------- | ------------------------------------------------------------ | ---------------------------------------------- |
+| Policy only                       | An etcd3 or Kubernetes datastore | Use {{site.prodname}} network policy to control firewalls on non-cluster hosts. | Binary install with/ without a package manager |
+| Networking and network policy     | An etcd3 datastore               | **Networking**<br />Use {{site.prodname}} networking (BGP, or overlay with VXLAN or IP-in-IP) to handle these communications:<br />- pod ↔ pod<br />- pod ↔ host<br /><br />**Note**: {{site.prodname}} does not handle host ↔ host networking; your underlying network must already be set up to handle this. <br /><br />**Policy**<br />Use {{site.prodname}} network policy to control firewalls on your non-cluster hosts. | Docker container                               |
 
-| Communication type   | Handled by         |
-|----------------------|--------------------|
-| Workload ↔ Workload  | {{site.prodname}}  |
-| Workload ↔ Host      | {{site.prodname}}  |
-| Host ↔ Host          | Underlying network |
+### Before you begin
 
-This means that if you are using {{site.prodname}} networking in your cluster, and you want the non-cluster hosts to be able to communicate with workloads in the cluster, you will need to install {{site.prodname}} for networking on your non-cluster hosts.  If you are not using {{site.prodname}} for networking in your cluster (e.g. are using {{site.prodname}} in policy-only mode), or don't need your non-cluster hosts to communicate directly with workloads, you can install {{site.prodname}} in policy-only mode on your non-cluster hosts.
+**Supported**
 
-#### {{site.prodname}} policy on non-cluster hosts
+- All platforms in this release, except Windows  
 
-{{site.prodname}} policy allows you to control firewalls on your non-cluster hosts using the same controls powerful controls as in your cluster.  {{site.prodname}} must be running on each non-cluster host you want to control.
+**Required**
 
-Using {{site.prodname}}, you can secure network interfaces of the host; these interfaces are called **host endpoints** (to distinguish them from **workload endpoints**). Host endpoints can have labels, which work the same as labels on workload endpoints. This allows you to create {{site.prodname}} network policy for either host or workload endpoints, where each selector can refer to the either type (or be a mix of the two) using labels.
+- Non-cluster host meets [system requirements](./requirements) for {{site.prodname}}. If you want to use a package manager for installation, the non-cluster host must be an Ubuntu- or RedHat-derived system.
```

Comment:
"Ubuntu or RedHat" instead of "Ubuntu- or RedHat-", I think?

Reply:
It is technically correct as is, but reworded to avoid clunkiness.